lib/b2b/organizations_members_oauth_providers.ts

```diff
@@ -0,0 +1,172 @@
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import {} from "../shared/method_options";
```

Comment:
Is this okay? If not we can figure out how to omit it.

Reply:
Should be fine